### PR TITLE
Use Lato as default font

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

**Comment by JDWarner** [comment]
No idea why Travis is failing with the following message after a very minor style change that doesn't touch any files flake8 should be worried about... but this exception is repeatable (I restarted that job, same result).

```python
**************************************************
*                  Flake8 test                   *
**************************************************
+flake8 --exit-zero '--exclude=test_*,six.py' skimage doc/examples viewer_examples
Traceback (most recent call last):
  File "/home/travis/venv/bin/flake8", line 7, in <module>
    from flake8.main import main
  File "/home/travis/venv/local/lib/python2.7/site-packages/flake8/main.py", line 5, in <module>
    import setuptools
  File "/home/travis/venv/local/lib/python2.7/site-packages/setuptools/__init__.py", line 11, in <module>
    from setuptools.extension import Extension
  File "/home/travis/venv/local/lib/python2.7/site-packages/setuptools/extension.py", line 8, in <module>
    from .dist import _get_unpatched
  File "/home/travis/venv/local/lib/python2.7/site-packages/setuptools/dist.py", line 21, in <module>
    packaging = pkg_resources.packaging
AttributeError: 'module' object has no attribute 'packaging'
```

**Comment by blink1073** [comment]
Ooooh nooo!  I messed up by merging #1426.  I thought it was failing coveralls when I merged it, but it was failing Travis.

**Comment by stefanv** [comment]
What have you done!?


:)​

**Comment by blink1073** [comment]
I'm working on a fix now.  :sheep:

**Comment by blink1073** [comment]
Should be fixed in #1452, it got past the failure point on my local build.

**Comment by JDWarner** [comment]
Build restarted.

File: doc/source/themes/scikit-image/layout.html [hunk 1]
```diff
@@ -86,7 +86,7 @@
             <ul class="nav">
                 {% include 'navbar.html' %}
             </ul>
-            <form class="navbar-form pull-right" action="{{ pathto('search') }}" method="get">
+            <form class="navbar-form pull-right" action="/docs/dev/search.html" method="get">
```
Comment (tacaswell) [inline doc/source/themes/scikit-image/layout.html]: Did you mean to hard-code this?

File: doc/source/themes/scikit-image/layout.html [hunk 2]
```diff
@@ -36,8 +35,9 @@
 {%- macro css() %}
     <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}" type="text/css" />
     <link href="{{ pathto('_static/', 1) }}css/bootstrap.min.css" rel="stylesheet" type="text/css">
-    <link href="{{ pathto('_static/', 1) }}css/custom.css" rel="stylesheet" type="text/css">
-    <link href="http://fonts.googleapis.com/css?family=Raleway" rel="stylesheet" type="text/css">
+    <link href="{{ pathto('_static/', 1) }}css/custom.css"
+    rel="stylesheet" type="text/css">
```
Comment (tacaswell) [inline doc/source/themes/scikit-image/layout.html]: line break?

**Pull request closed without merging**